websites/W/Wbijam/iframe.ts

```diff
@@ -0,0 +1,17 @@
+const iframe = new iFrame();
+iframe.on("UpdateData", async () => {
+	let video: HTMLVideoElement;
+
+	if (document.querySelector("video")) video = document.querySelector("video");
```

Comment:
```suggestion
	if (document.querySelector("video")) const video = document.querySelector("video");
```

Reply:
![image](https://user-images.githubusercontent.com/63014799/180400270-bc59aa92-1d6f-4054-953a-a6425ca22f2b.png)

Reply:
This can be ignored

Reply:
this error or this suggestion?

Reply:
Error

Reply:
so if you approved changes, i need to change it?

Reply:
You may leave it